specification/resources/resource-manager/Microsoft.Authorization/2017-06-01-preview/policy.json

```diff
@@ -0,0 +1,1111 @@
+        },
+        "metadata": {
+          "type": "object",
+          "description": "The policy definition metadata."
+        },
+        "parameters": {
+          "type": "object",
+          "description": "Required if a parameter is used in policy rule."
+        }
+      },
+      "description": "The policy definition properties."
+    },
+    "PolicySetDefinitionProperties": {
+      "properties": {
+        "policyType": {
+          "type": "string",
+          "description": "The type of policy definition. Possible values are NotSpecified, BuiltIn, and Custom.",
+          "enum": [
+            "NotSpecified",
+            "BuiltIn",
+            "Custom"
+          ],
+          "x-ms-enum": {
+            "name": "policyType",
+            "modelAsString": true
+          }
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the policy set definition."
+        },
+        "description": {
+          "type": "string",
+          "description": "The policy set definition description."
+        },
+        "metadata": {
+          "type": "object",
+          "description": "The policy set definition metadata."
+        },
+        "parameters": {
+          "type": "object",
+          "description": "The policy set definition parameters that can be used in policy definition references."
+        },
+        "policyDefinitions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PolicyDefinitionReference"
+          },
+          "description": "An array of policy definition references."
+        }
+      },
+      "required": [
+        "policyDefinitions"
+      ],
+      "description": "The policy set definition properties."
+    },
+    "PolicyDefinitionReference": {
+      "properties": {
+        "policyDefinitionId": {
+          "type": "string",
+          "description": "The ID of the policy definition or policy set definition."
+        },
+        "parameters": {
+          "type": "object",
+          "description": "Required if a parameter is used in policy rule."
+        }
+      },
+      "description": "The policy definition reference."
+    },
+    "PolicyDefinition": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyDefinitionProperties",
+          "description": "The policy definition properties."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The ID of the policy definition."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the policy definition. If you do not specify a value for name, the value is inferred from the name value in the request URI."
+        }
+      },
+      "description": "The policy definition.",
+      "x-ms-azure-resource": true
+    },
+    "PolicySetDefinition": {
```

Comment:
Seems like this model is missing property named as `type` from the example.

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/272643689#L691